src/utils/components/NetworkInterfaceModal/utils/helpers.ts

```diff
@@ -46,6 +50,15 @@ export const updateVMNetworkInterfaces = (
         path: '/spec/template/spec/domain/devices/interfaces',
         value: updatedInterfaces,
       },
+      ...(isEmpty(updatedInterfaces)
+        ? [
+            {
+              op: getAutoAttachPodInterface(vm) === undefined ? 'add' : 'replace',
```

Comment:
there are new K8S ops const for ad, replace , @pcbailey please show him where

Reply:
Ah yes right. I'll fix that